src/components/footer/_macro.njk

```diff
@@ -158,9 +158,11 @@
                             </div>
                         {% endif %}
                         {% if not params.poweredBy %}
-                            <div class="ons-footer__poweredby ons-u-mb-m">
-                                {{ onsLogo | safe }}
-                            </div>
+                            <a class="ons-footer__poweredByLink" {% if lang == "cy" %}href="https://cy.ons.gov.uk/"{% else %}href="https://www.ons.gov.uk/"{% endif %}>
+                                <div class="ons-footer__poweredby ons-u-mb-m">
+                                    {{ onsLogo | safe }}
+                                </div>
+                            </a>
```

Comment:
Is this something we want to make mandatory? And the link not configurable by the user like we have in other places? I get that its always the same logo but I was expecting this to be controlled by params.
Feels like something we may need to have a discussion on with the rest of the team, I just wanted to check to see if there were any use cases for when a service might not want this

Reply:
I think if the user uses the powered by parameter they can pass their html with their links. If we had a parameter that allowed you to pass in a Logo and we managed the layout and positioning of it like the header that we would also give the url parameter. But in this case it does look like the only icons that are made available in the footer are chosen by us so we might as well wrap them with the correct link. But maybe we can talk about making the footer's logo placements more aligned with the header component? @balibirchlee I think this is what we discussed right? It would be a separate ticket to improve our docs and code to fully manage the logos placements in the footer.

Reply:
Yeah fair enough, I just thought it would offer the user more flexibility but mainly I just wanted to make sure there wasn't a use case for it or not and it wouldn't be much work to parametrise it as it will enforce this change on every service.

Reply:
sorry for delay on this - illness etc.

So this is covered in the text under 'What is the context of this PR?' - I'm happy to create a ticket for it, so it doesn't get lost?